python/Ganga/GPIDev/Lib/Tasks/ITransform.py

```diff
@@ -27,7 +27,6 @@ class ITransform(GangaObject):
         'unit_merger': ComponentItem('mergers', defvalue=None, load_default=0, optional=1, doc='Merger to be copied and run on each unit separately.'),
         'copy_output': ComponentItem('datasets', defvalue=None, load_default=0, optional=1, doc='The dataset to copy all units output to, e.g. Grid dataset -> Local Dataset'),
         'unit_copy_output': ComponentItem('datasets', defvalue=None, load_default=0, optional=1, doc='The dataset to copy each individual unit output to, e.g. Grid dataset -> Local Dataset'),
-        'unit_merger': ComponentItem('mergers', defvalue=None, load_default=0, optional=1, doc='Merger to be run copied and run on each unit separately.'),
```

Comment:
In the spirit of this pull request (**PR**), namely to illustrate the powerful code reviewing tools available through github, I though I'd show people what an in-line code comment looks like. What follows is a stupid I'm sure question...sorry Matt.

@milliams the title of the commit that this change comes from bc7d4d6cc9b281cdbccd6492083900ff11c2938d is `Remove duplicate dictionary keys` however I don't see the duplicate in this case. Do we definitely want to remove the unit_merger attribute?